5.10.rc7 is failing to build

I've built every other release candidate for 5.10 perfectly fine with this PKGBUILD, and I didn't change anything in the config. No additional patches, just fsync enabled, regular cfs scheduler, etc. ACS patch is off, bcachefs is off, Anbox modules off, basically everything pretty much vanilla. 

This is the error, and it's all there is throughout the scrollback limit of my terminal (which is like 10000 lines or something crazy, I set it really high a while back). 

```
FAILED unresolved symbol __add_to_page_cache_locked
make: *** [Makefile:1170: vmlinux] Error 255
  DESCEND  objtool
  DESCEND  bpf/resolve_btfids
  CALL    scripts/atomic/check-atomics.sh
  CALL    scripts/checksyscalls.sh
  CHK     include/generated/compile.h
  GEN     .version
  CHK     include/generated/compile.h
  UPD     include/generated/compile.h
  CC      init/version.o
  AR      init/built-in.a
  LD      vmlinux.o
  MODPOST vmlinux.symvers
  MODINFO modules.builtin.modinfo
  GEN     modules.builtin
  LD      .tmp_vmlinux.btf
  BTF     .btf.vmlinux.bin.o
  LD      .tmp_vmlinux.kallsyms1
  KSYMS   .tmp_vmlinux.kallsyms1.S
  AS      .tmp_vmlinux.kallsyms1.S
  LD      .tmp_vmlinux.kallsyms2
  KSYMS   .tmp_vmlinux.kallsyms2.S
  AS      .tmp_vmlinux.kallsyms2.S
  LD      vmlinux
  BTFIDS  vmlinux
FAILED unresolved symbol __add_to_page_cache_locked
make: *** [Makefile:1170: vmlinux] Error 255
==> ERROR: A failure occurred in build().
    Aborting...
```

Happy to provide any more info as needed.  

Actually, I just tried running it again, with fsync set to false, just to be sure, and it still fails in the exact same spot with the exact same error. It's weird, because it's seemingly at/near the very end of the process

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -316,6 +316,7 @@ case $_basever in
         0011-ZFS-fix.patch
         #0012-linux-hardened.patch
         0012-misc-additions.patch
+        0013-v5.10-rc7-revert.patch
     )
     sha256sums=('9f95194fc84eef01789f2ed6566518ef597e9c6541b640f7276f2a3ef1a221f2'
             'SKIP'
@@ -339,7 +340,8 @@ case $_basever in
             'a557b342111849a5f920bbe1c129f3ff1fc1eff62c6bd6685e0972fc88e39911'
             'a5149d7220457d30e03e6999f35a050bce46acafc6230bfe6b4d4994c523516d'
             '49262ce4a8089fa70275aad742fc914baa28d9c384f710c9a62f64796d13e104'
-            '433b919e6a0be26784fb4304c43b1811a28f12ad3de9e26c0af827f64c0c316e')
+            '433b919e6a0be26784fb4304c43b1811a28f12ad3de9e26c0af827f64c0c316e'
+            '748f7d9db58946d82caf4fe1c76d4f855eee806aa140b0aa69236f1f89a3e5c6')
 	;;
 esac
 

--- a/linux-tkg-config/prepare
+++ b/linux-tkg-config/prepare
@@ -363,6 +363,11 @@ _tkg_srcprep() {
     tkgpatch="$srcdir/0012-misc-additions.patch" && _tkg_patcher
   fi
 
+  if [ "$ver510" = "rc7" ] && [ "$_basever" = "510" ]; then
+    msg2 "Applying 5.10-rc7 compilation patch"
+    tkgpatch="$srcdir/0013-v5.10-rc7-revert.patch" && _tkg_patcher
+  fi
+
   # prjc/bmq patch rev
   if [ "$_basever" = "58" ] || [ "$_basever" = "57" ]; then
     rev=3

--- a/linux-tkg-patches/5.10/0013-v5.10-rc7-revert.patch
+++ b/linux-tkg-patches/5.10/0013-v5.10-rc7-revert.patch
@@ -1,0 +1,11 @@
+--- a/mm/filemap.c	2020-12-07 01:16:40.233503592 -0800
++++ b/mm/filemap.c	2020-12-07 01:16:05.211387455 -0800
+@@ -827,7 +827,7 @@
+ }
+ EXPORT_SYMBOL_GPL(replace_page_cache_page);
+ 
+-static noinline int __add_to_page_cache_locked(struct page *page,
++noinline int __add_to_page_cache_locked(struct page *page,
+ 					struct address_space *mapping,
+ 					pgoff_t offset, gfp_t gfp,
+ 					void **shadowp)

